components/popUp/popUp.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./popUp.module.css";
+
+export type PopUpProps = {
+  label: string;
+  content: string;
+  handleClose: (event: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
events in properties should start with `on` to have similar naming convention like existing React components.
In addition, Storybook can automatically log actions starting with `on`:
https://storybook.js.org/docs/react/essentials/actions#automatically-matching-args

```suggestion
  onClose: (event: React.MouseEvent<HTMLButtonElement>) => void;
```